src/assets/items.json

```diff
@@ -0,0 +1,74 @@
+[
+  {
+    "id": "CMDIRT",
+    "name": "Common Dirt",
+    "desc": "Just some dirt, no big deal.",
+    "amount": 0,
```

Comment:
is this a definition of types of materials or an inventory or both?

amount seems like it doesnt belong in the same object as a description

Reply:
the _intention_ was to make these item definitions, but looking through how my item inventory is implemented, i  think they're pulling double-duty, which is dumb. i'll refactor that in another PR, when i get around to acquiring items from digging. right now they're not really being used.